server/sonar-web/src/main/js/apps/issues/conciseIssuesList/__tests__/__snapshots__/ConciseIssueLocations-test.js.snap

```diff
@@ -0,0 +1,27 @@
+exports[`test should render one flow 1`] = `
+<div
+  className="pull-right">
+  <ConciseIssueLocationBadge
+    count={3} />
+</div>
+`;
+
+exports[`test should render only secondary locations 1`] = `
+<div
+  className="pull-right">
+  <ConciseIssueLocationBadge
```

Comment:
Is this result really correct ? Or should it be 3 `ConciseIssueLocationBadge` with `count = 1` ?